pages/common/ssh-keygen.md

```diff
@@ -33,3 +29,7 @@
 - Change the password of a key:
 
 `ssh-keygen -p -f ~/.ssh/{{filename}}`
```

Comment:
All the path is the argument option?
```suggestion
`ssh-keygen -p -f {{~/.ssh/filename}}`
```

Reply:
Definitely.

Reply:
yes, all the path is the argument and it defaults to `~/.ssh/id_rsa` if not specified  

most of the operations that require an -f (file path) default to  `~/.ssh/id_rsa` or  `~/.ssh/id_rsa.pub`